__tests__/app.test.js

```diff
@@ -72,6 +72,7 @@ describe("/api/reviews/:review_id", () => {
           category: "social deduction",
           owner: "bainesface",
           created_at: "2021-01-18T10:01:41.251Z",
+          comment_count: 3,

```

Comment:
Ideally, you shouldn't need to edit a past test. I would advice using .toMatchObject in future as it won't fail if you add any extra properties. For now though, with .toEqual, I understand this would have failed otherwise